From randy.dunlap@oracle.com  Tue Feb 17 17:39:38 2009
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Mon, 16 Feb 2009 08:46:16 -0800
Subject: Staging: phison: fix kconfig for clean build
To: Stephen Rothwell <sfr@canb.auug.org.au>
Cc: Greg KH <gregkh@suse.de>, Andrew Morton <akpm@linux-foundation.org>
Message-ID: <49999858.6000604@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

phison is an ATA driver, not a classic IDE driver, so fix the Kconfig file
so that it will build.

drivers/staging/phison/phison.c:43: error: implicit declaration of function 'ATA_BMDMA_SHT'
drivers/staging/phison/phison.c:43: error: initializer element is not constant
drivers/staging/phison/phison.c:43: error: (near initialization for 'phison_sht.module')
drivers/staging/phison/phison.c:47: error: 'ata_bmdma_port_ops' undeclared here (not in a function)
drivers/staging/phison/phison.c:65: error: implicit declaration of function 'ata_pci_sff_init_one'

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/phison/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/phison/Kconfig
+++ b/drivers/staging/phison/Kconfig
@@ -1,5 +1,5 @@
 config IDE_PHISON
 	tristate "PCIE ATA PS5000 IDE support"
-	depends on IDE
+	depends on ATA && ATA_SFF
 	---help---
 	  This is an experimental driver for PS5000 IDE driver.
